sccameraview/src/main/java/com/example/sccameraview/Camera2View.java

```diff
@@ -355,6 +366,41 @@ private static Size chooseVideoSize(Size[] choices) {
         return choices[choices.length - 1];
     }
 
+    private File getOutputMediaFile(int type){
```

Comment:
Why do we need this method exaclty? The client should be able to decide where to save the file.